frontend/src/components/ExpensePage.js

```diff
@@ -1,21 +1,50 @@
-import React from "react";
+import React, { Component } from "react";
 import Header from "./Header";
 import ExpensePresentation from "./ExpensePresentation";
 import ExpenseList from "./ExpenseList";
 import Footer from "./Footer";
+
 import "../styles/ExpensePage.css";
 
-function ExpensePage() {
-  return (
-    <div className="ExpensePage">
-      <Header />
-      <main>
-        <ExpensePresentation />
-        <ExpenseList />
-      </main>
-      <Footer />
-    </div>
-  );
+export class ExpensePage extends Component {
+  state = {
+    total: 0,
+  };
+
+  updateTotal = (amount) => {
+    this.setState({ total: this.state.total + parseFloat(amount) });
+  };
+
+  // React Life Cycle
+  componentDidMount() {
+    this.total = JSON.parse(localStorage.getItem("total"));
+    if (localStorage.getItem("total")) {
+      this.setState({
+        total: this.total,
+      });
+    } else {
+      this.setState({
+        total: 0,
+      });
+    }
```

Comment:
This could also be cleaned a bit, e.g. the whole function could be:
```js
  componentDidMount() {
    let lsTotal = localStorage.getItem("total");
    lsTotal
        ? this.setState({ total: JSON.parse(lsTotal) })
        : this.setState({ total: 0 });
  }
```

Reply:
Cleaned this code.